werkzeug/datastructures.py

```diff
@@ -2559,12 +2559,12 @@ def mimetype(self):
         """Like :attr:`content_type` but without parameters (eg, without
         charset, type etc.).  For example if the content
```

Comment:
This is a bit hard to read. I recommend: "Like ..., but without parameters (...) and always lowercase."